Docs/contributing.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to Vapor
```

Comment:
We should start fresh here. The current `contributing.md` in Vapor needs a bit of help and is mostly outdated.

Reply:
Updated.